### ISSUE TITLE
Will not download 1080p version of the file

### ISSUE THREAD

**Issue by numlockkey** [FaisalUmair/udemy-downloader-gui#314]
I set it to download the highest video quality and it only downloads the 720p version even though 1080p is available, as I was able to verify it with Ant video downloader.

The course I was trying to download is this:
https://www.udemy.com/course/crash-course-electronics-and-pcb-design/

**Comment by FaisalUmair** [comment]
Yes I am aware that Udeler is only able to download 720p videos.  I will keep this issue open until I fix it.

**Comment by kazepis** [comment]
@FaisalUmair great tool! Keep up the excellent work! Do you have any update on this issue?

**Comment by Snipe3000** [comment]
Im having this issue as well

**Comment by satvikpendem** [comment]
Same here

**Comment by ronmat** [comment]
As of 1.8.2 this appears to be working. During debugging I noticed that a lot of the videos but not all in a course were 720p. I console printed the video list and some were 1080p. Set the quality to max and started watching and when it got to the 1080p videos the quality icon changed and the downloaded video was 1080p. If you change the settings you need to restart the app. I also set it to 480p and restarted the program and all the videos downloaded as 480p. So at least mine appears to be working fine.

**Comment by mostlyhuman** [comment]
Hmm, definitely not working for me in 1.8.2, everything is still capped at 720p.  I have tried both 1080p and Highest and I did restart the app each time after changing and saving the setting.
